bundle/config/mutator/resourcemutator/run_as.go

```diff
@@ -129,6 +129,16 @@ func validateRunAs(b *bundle.Bundle) diag.Diagnostics {
 		))
 	}
 
+	// Secret scopes do not support run_as in the API.
```

Comment:
Did we check with the team?  For secret scopes it seems we should add them to allowlist for run_as since they are not tied to compute.